bundles/org.openhab.binding.nuvo/README.md

```diff
@@ -0,0 +1,394 @@
+Up to 20 keypad zones can be controlled when zone expansion modules are used (if not all zones on the amp are used they can be excluded via configuration).
+
+The binding supports two different kinds of connections:
+
+* serial connection,
+* serial over IP connection
+
+For users without serial connector on server side, you can use a serial to USB adapter.
+
+You don't need to have your Grand Concerto or Essentia G whole house amplifier device directly connected to your openHAB server.
+You can connect it for example to a Raspberry Pi and use [ser2net Linux tool](https://sourceforge.net/projects/ser2net/) to make the serial connection available on LAN (serial over IP).
+
+## Supported Things
+
+There is exactly one supported thing type, which represents the amplifier controller.
+It has the `amplifier` id.
+
+## Discovery
+
+Discovery is not supported.
+You have to add all things manually.
+
+## Binding Configuration
+
+There are no overall binding configuration settings that need to be set.
+All settings are through thing configuration parameters.
+
+## Thing Configuration
+
+The thing has the following configuration parameters:
+
+| Parameter Label         | Parameter ID | Description                                                                                                                        | Accepted values          |
+|-------------------------|--------------|------------------------------------------------------------------------------------------------------------------------------------|--------------------------|
+| Serial Port             | serialPort   | Serial port to use for connecting to the Nuvo whole house amplifier device                                                         | a comm port name       | |
+| Address                 | host         | Host name or IP address of the machine connected to the Nuvo whole house amplifier device (serial over IP)                         | host name or ip        | |
+| Port                    | port         | Communication port (serial over IP).                                                                                               | ip port number         | |
+| Number of Zones         | numZones     | (Optional) Number of zones on the amplifier to utilize in the binding (up to 20 zones when zone expansion modules are used)        | (1-20; default 6)      | |
+| Sync Clock on GConcerto | clockSync    | (Optional) If set to true, the binding will sync the internal clock on the Grand Concerto to match the openHAB host's system clock | Boolean; default false | |
+
+Some notes:
+
+* If a zone has a maximum volume limit configured by the Nuvo configurator, the volume slider will automatically drop back to that level if set above the configured limit.
+* Source display_line1 thru 4 can only be updated on non NuvoNet sources.
+* The track_position channel does not update continuously for NuvoNet sources. It only changes when the track changes or playback is paused/unpaused.
+*
```

Comment:
wanted to do a line break, but that does not work in markdown. removed.

Reply:
In normal text, you can add two spaces at the end of the line to insert a line break. Maybe that helps here, too.

Reply:
I tried that but did it did not work since it is part of a list. It will be fine here without a line break. I did make improvements to all my bindings' markdown.